client/src/pages/workspace/mutableInput/MutableInput.component.tsx

```diff
@@ -0,0 +1,60 @@
+import React, { useState, useRef, useEffect, ChangeEvent, KeyboardEvent } from "react";
+import OutsideClickHandler from 'react-outside-click-handler';
+
+import "./MutableInput.style.css";
+import {MutableInputProps} from "./props";
+
+const MutableInput: React.FC<MutableInputProps> = ({ text, textSet, placeHolder }) => {
```

Comment:
name of the file and name of the component are different

Reply:
?

Reply:
File name : MutableInput.component
Component name : MutableInput

Reply:
You sill did not fix it